instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
@smarnach I replied to your question on the JIRA ticket but I'll include it here as well:

It *should* be possible to set the default value of `DEFAULT_FILE_STORAGE` in the configuration repo to null. This should cause both the `aws.py` and the `openstack.py` settings files to use the correct storage backend as a fallback.

The downside is that for deployments that use the default settings and include AWS credentials, this will change the storage backend from the file system to S3. This will break backwards compatibility, and upstream may not be happy with it. Since upstream have already taken ages to review the PR I'm reluctant to add any more breaking changes to it.

What we can do for the instance manager is explicitly set `DEFAULT_FILE_STORAGE` to null. Would that work?